src/main/resources/application.yaml

```diff
@@ -1,4 +1,19 @@
+#Adds the option to go to eg. http://localhost:8080/actuator/health for seeing the running configuration
```

Comment:
Did you review each of the changes here and apply the ones that make sense to our custom configuration file?